File: app/src/main/java/eu/caraus/dynamo/application/domain/udacity/UdacityCourses.java
package eu.caraus.dynamo.application.domain.udacity;

import java.io.Serializable;
import java.util.List;

import com.google.gson.annotations.SerializedName;


public class UdacityCourses implements Serializable {

	@SerializedName("courses")
	private List<CoursesItem> courses;

	public void setCourses(List<CoursesItem> courses){
		this.courses = courses;
	}

	public List<CoursesItem> getCourses(){
		return courses;
	}

	@Override
 	public String toString(){
		return 
			"UdacityCourses{" +
			"courses = '" + courses + '\'' + 
			"}";
		}
}

File: app/src/main/java/eu/caraus/dynamo/application/data/domain/udacity/UdacityCourses.kt
package eu.caraus.dynamo.application.data.domain.udacity

import java.io.Serializable

import com.google.gson.annotations.SerializedName



data class UdacityCourses (

    @SerializedName("courses")
    var courses : List<CoursesItem>

) : Serializable